code/modules/point/point.dm

```diff
@@ -122,7 +122,11 @@
 
 		if(!(pointing_at in view(client.view, src)))
 			return FALSE
-
+	if(iscarbon(src)) // special interactions for carbons
+		var/mob/living/carbon/our_carbon = src
+		if(our_carbon.usable_hands <= 0 || HAS_TRAIT(src, TRAIT_HANDS_BLOCKED))
+			our_carbon.emote("point [pointing_at]")
```

Comment:
"Mothblocks point the floor"?

```suggestion
			our_carbon.emote("points at [pointing_at]")
```

Reply:
![image](https://github.com/user-attachments/assets/a4a96494-07b4-4868-9d57-02e1346084ad)

with current code

Reply:
because
```php
/datum/emote/living/point
	key = "point"
	key_third_person = "points"
	message = "points."
	message_param = "points at %t."
	cooldown = 3 SECONDS
```

Reply:
huh.